### PR TITLE
Rsa algorithm

### PR DESCRIPTION
**Pull request by rohangawhade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
### Description

Added RSA Algorithm implementation using python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1222

### CODE REVIEW

File: Python/cryptography/RSA_Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import math
```
Comment (iamrajiv) [inline Python/cryptography/RSA_Algorithm.py:1]: @rohangawhade Add little bit description about RSA algorithm.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/cryptography/RSA_Algorithm.py (hunk 1 above)
Comment (rohangawhade) [reply Python/cryptography/RSA_Algorithm.py:1]: @iamrajiv Added the description about RSA Algorithm.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@rohangawhade Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Comment by rohangawhade** [comment]
> @rohangawhade Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

@iamrajiv  The code is already formatted according to PEP 8 rules.

File: Python/cryptography/RSA_Algorithm.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    for i in range(1, m):
+        if(p * i) % m == 1:
+            return i
+    return 1
+
+if __name__ == '__main__':
+    '''p and q  are prime numbers. pt is plain text.
+    e is derived number which lies between 1 and (p-1)*(q-1)
+    '''
+    p, q, e, pt = [int(x) for x in input("Enter p, q, e and plaintext: ").split()]
```
Comment (iamrajiv) [inline Python/cryptography/RSA_Algorithm.py]: @rohangawhade line too long (82 > 79 characters)

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/cryptography/RSA_Algorithm.py (hunk 2 above)
Comment (rohangawhade) [reply Python/cryptography/RSA_Algorithm.py]: @iamrajiv  done.

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**